tools/install.sh

```diff
@@ -336,6 +344,18 @@ elif [ "$LINUX_VERSION" == "BackBox" ] || \
 		easy_install -U distribute
 		apt-get install -y postgresql-server-dev-all &> /dev/null
 	fi
+elif [ "$LINUX_VERSION" == "Arch" ]; then
+	pacman --noconfirm -S freetype2 python python-pip pkg-config
```

Comment:
That is installing Python3. Python3 is the default Python for Arch (which is why just python is used -- to target Python2 would be python2).